src/hapi.js

```diff
@@ -16,9 +16,9 @@ function registerRoutes(app, { routes : r }, callback) {
 
     routes.unshift(healthcheckRoute);
 
```

Comment:
I was gonna add that same logic to Express, but I looked up the documentation, and extra parameters are assumed to be extra callback functions.